homeassistant/components/braviatv/const.py

```diff
@@ -6,7 +6,7 @@
 
 CONF_IGNORED_SOURCES = "ignored_sources"
 
-BRAVIARC = "braviarc"
+BRAVIA_CLIENT = "braviaclient"
```

Comment:
Please separate words with underscore in variable names etc.
```suggestion
BRAVIA_CLIENT = "bravia_client"
```